scripts/install.sh

```diff
@@ -96,6 +96,8 @@ elif [ "$uname_machine" = "armv7" ] || [ "$uname_machine" = "armv7l" ]; then
 # armv8?
 elif [ "$uname_machine" = "arm64" ]; then
   arch="arm64"
+elif [ "$uname_machine" = "aarch64" ]; then
+  arch="arm64"
```

Comment:
Can you combine these two lines and squash into one commit?
```suggestion
elif [ "$uname_machine" = "arm64" ] || [ "$uname_machine" = "aarch64" ]; then
  arch="arm64"
```

Reply:
I unfortunately can't get my squash working, but I have combined the two lines and added chore: to the beginning of my last commit message.

Reply:
You can undo your 3 commits with `git reset HEAD~3` and then re-add your changes in one commit. We try really hard to keep a clean commit history in this repo, in part because the version changelogs are generated directly from the commit messages.

Reply:
Perfect, I fixed it. Thank you